x/evm/keeper/state_transition.go

```diff
@@ -68,7 +68,14 @@ func (k *Keeper) NewEVM(
 		tracer = k.Tracer(ctx, msg, cfg.ChainConfig)
 	}
 	vmConfig := k.VMConfig(ctx, msg, cfg, tracer)
-	return k.evmConstructor(blockCtx, txCtx, stateDB, cfg.ChainConfig, vmConfig)
+
+	enabledPrecompiles := k.GetParams(ctx).EnabledPrecompiles
```

Comment:
Params is in EVMConfig so we don't need to fetch it again - `cfg.Params.EnabledPrecompiles`

Reply:
got it